scripts/qa_crowbarsetup.sh

```diff
@@ -3810,6 +3819,33 @@ function oncontroller_testsetup()
     nova stop "$instanceid"
     wait_for 100 1 "test \"x\$(nova show \"$instanceid\" | perl -ne 'm/ status [ |]*([a-zA-Z]+)/ && print \$1')\" == xSHUTOFF" "testvm to stop"
 
+    # run tests for Magnum bay deployment
```

Comment:
According to source code of functional tests for Magnum bay_tests are only for `swarm` containers orchestrator engine, even for master branch
```
https://github.com/openstack/magnum/blob/stable/mitaka/magnum/tests/functional/api/v1/test_bay.py#L61
```

We have to stay with our integration tests.